soh/src/code/z_onepointdemo.c

```diff
@@ -69,7 +69,30 @@ s32 OnePointCutscene_SetInfo(PlayState* play, s16 camIdx, s16 csId, Actor* actor
     PosRot sp8C;
     f32 tempRand;
     Unique9OnePointCs* csInfo = ONEPOINT_CS_INFO(csCam);
+    
+    //the default is 90, lower values necessary to prevent camera swing as animation speeds up
+    //arrived at via testing camera movement when holding left/right during the animation
+    s16 camCrawlTemp = CVarGetInteger("gCrawlSpeed", 0);
+    s16 camCrawlTimer;
 
+    switch (camCrawlTemp) { 
+        case 1:
+            camCrawlTimer = 90;
+            break;
+        case 2:
+            camCrawlTimer = 50;
+            break;
+        case 3:
+            camCrawlTimer = 38;
+            break;
+        case 4:
+            camCrawlTimer = 31;
+            break;
+        case 5:
+            camCrawlTimer = 26;
+            break;
+    }
```

Comment:
I highly recommend adding a default: here that sets it to 90.

Reply:
Done!